core-common/src/main/java/org/zalando/nakadi/repository/kafka/KafkaSettings.java

```diff
@@ -26,7 +28,11 @@ public class KafkaSettings {
     private final String clientRack;
     private final String compressionType;
     private final int socketSendBufferBytes;
-
+    private final Optional<Integer> preferredListenerPort;
```

Comment:
maybe lets call it sslPort, I assume that the whole purpose of the change and it is correlated with other params.

Reply:
or some other name which indicates relation to ssl config

Reply:
I hope that this configuration value will be used only for a listener that has some security, but in the notion of configuring the Kafka client it is still only a port, therefore I think it is more precise to not specify what kind of security is configured on the Broker side